plugins/Unity/Launcher/xdgwatcher.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2019 UBports project.
```

Comment:
If copyright is being assigned to the foundation, I think we should use the Foundation's legal name here (and anywhere else it's used like this).

Reply:
Cant use that yet as foundation is still not full up.

Reply:
I thought legal status had been granted?

Reply:
So, "UBports project" is not a legal entity which can have Copyright assigned to it. It either needs to be assigned to `UBports Foundation` or otherwise should say `Marius Gripsgard` given you are the author and thus retain the copyright (unless you've copied code from elsewhere that is under a suitable license where you can release it as GPLv3).

Reply:
Now that the Foundation is up, i  edited this.